mindmeld/active_learning/classifiers.py

```diff
@@ -183,6 +248,13 @@ def _get_probs(
         Returns:
             prob_vector (List[List]]): Probability distribution vectors for given queries.
         """
+        if nlp_component_type == "entity":
```

Comment:
Since we're using nlp_component_type as a param for this function it would be more consistent to also add another type "classifier", this can be the default for the function

Reply:
What would this additional type be used for?

Reply:
Added equivalent methods for classifier and tagger.